### PR TITLE
Prevent removing items from house if player is not invited

### PR DESCRIPTION
**Pull request by nekiro**
You shouldn't be able to remove items from house if you are not invited (open door scenario).

### CODE REVIEW

**Review by DSpeichert — DISMISSED** [review]

**Comment by ranisalt** [comment]
How can you move items inside a house you are not invited to? How do you get inside it to begin with?

**Comment by nekiro** [comment]
> How can you move items inside a house you are not invited to? How do you get inside it, to begin with?

Open door

**Review by mattyx14 — DISMISSED** [review]

**Comment by ranisalt** [comment]
> Open door

As far as I'm concerned even with an open door you can not enter a house if not invited.

**Comment by EPuncker** [comment]
> > Open door
> 
> As far as I'm concerned even with an open door you can not enter a house if not invited.

I think nekiro is talking about this scenario:

**Comment by ranisalt** [comment]
Ah, I see, that's clear to me now, sorry for the confusion.

**Review by EvilHero90 — APPROVED** [review]

**Review by EPuncker — DISMISSED** [review]

**Comment by soul4soul** [comment]
In cipbia I think you are allowed to move items from an open door. The exception is tile carpets. I recently remember picking up vials from an open door in cipbia.

**Comment by nekiro** [comment]
> In cipbia I think you are allowed to move items from an open door. The exception is tile carpets. I recently remember picking up vials from an open door in cipbia.

Yeah, checked it today, you cannot move carpets, but vials or other stuff can be moved.

File: src/configmanager.cpp [hunk 1]
```diff
@@ -146,7 +146,7 @@ bool ConfigManager::load()
 	boolean[ONLINE_OFFLINE_CHARLIST] = getGlobalBoolean(L, "showOnlineStatusInCharlist", false);
 	boolean[YELL_ALLOW_PREMIUM] = getGlobalBoolean(L, "yellAlwaysAllowPremium", false);
 	boolean[FORCE_MONSTERTYPE_LOAD] = getGlobalBoolean(L, "forceMonsterTypesOnLoad", true);
-	boolean[HOUSE_OWNED_BY_ACCOUNT] = getGlobalBoolean(L, "houseOwnedByAccount", false);
+	boolean[ONLY_INVITED_CAN_MOVE_HOUSE_ITEMS] = getGlobalBoolean(L, "onlyInvitedCanMoveHouseItems", true);
```
Comment (EPuncker) [inline src/configmanager.cpp]: seems like a little mistake 😜

**Review by EPuncker — CHANGES_REQUESTED** [review]

File: src/configmanager.cpp (hunk 1 above)
Comment (nekiro) [reply src/configmanager.cpp]: You are looking at outdated changes

**Review by EPuncker — APPROVED** [review]

**Comment by Diego-OT** [comment]
There is a bug when player "Use with" item

https://gyazo.com/2d5ca5a042bbdd1c5191a1dde192fb44

**Comment by EPuncker** [comment]
@nekiro could you check this please? I can't right now

**Comment by nekiro** [comment]
@EPuncker no, it does not happen.

**Comment by felipepips** [comment]
> You shouldn't be able to remove items from house if you are not invited (open door scenario).

Hello. I did all the changes and now i actually cant move items from house tiles, but if this item is a container (bag, backpack) i can open and take items from inside.. any fix for that? Thanks

**Comment by Emilianenko** [comment]
> > You shouldn't be able to remove items from house if you are not invited (open door scenario).
> 
> Hello. I did all the changes and now i actually cant move items from house tiles, but if this item is a container (bag, backpack) i can open and take items from inside.. either if the item in door tile is something useable (food) i can use (eat).. any fix for that? Thanks

@nekiro @EPuncker

**Comment by EPuncker** [comment]
> > You shouldn't be able to remove items from house if you are not invited (open door scenario).
> 
> Hello. I did all the changes and now i actually cant move items from house tiles, but if this item is a container (bag, backpack) i can open and take items from inside.. either if the item in door tile is something useable (food) i can use (eat).. any fix for that? Thanks

hello, please open an issue so this can be taken a look in the future

**Pull request merged**